interfaces/coin-sig.repl

```diff
@@ -0,0 +1,54 @@
+; --------------------------------------------------------------------------
+; --------------------------------------------------------------------------
+; Using formal verification in Pact interfaces to create example coin.
+
+(begin-tx)
+
+(define-keyset "test-keyset" (sig-keyset))
+
+(module acct-module "test-keyset"
+  @doc "account schema and table module"
+
+  (defschema account
+
+    balance:integer
+    ks:keyset
+  )
+
+  (deftable accounts:{account})
+)
+
+(interface coin-sig
+
+  "Example Coin Contract abstract interface"
+
+  (use acct-module)
+
+  (defun transfer (from:string to:string amount:integer)
+    @doc   "Transfer money between accounts"
+    @model [(property (row-enforced accounts "ks" from))
```

Comment:
This model looks great :+1:

Reply:
Do we want to stick with this approach or move to capabilities version?